apps/coscrad-frontend/src/store/slices/resources/songs/thunks/index.ts

```diff
@@ -1 +1 @@
-export * from './fetchSongs';
+export * from './fetch-songs';
```

Comment:
we should kebab case **everything** for consistency, at least in the redux and resource component layers for now.

Reply:
I'll save this for last as it will be mostly noise.